### ISSUE TITLE
KvSource ETCD fails

### ISSUE THREAD

**Issue by cocap10** [containous/staert#15]
I tried to parse a KvSource working with ETCD using `Staert.LoadConfig()` (which calls `KvSource.Parse()`) 
It prints errors on maps, slices or arrays fields :
```shell
4 error(s) decoding:
* 'DefaultEntryPoints': source data must be an array or slice, got string
* 'EntryPoints' expected a map, got 'string'
* 'Etcd' expected a map, got 'string'
* 'Web' expected a map, got 'string'
``` 

Probably due to the KV Store which stores a key/value for each directory :cry:

**Comment by emilevauge** [comment]
Deal breaker?

**Comment by cocap10** [comment]
closed by #16

**Comment by cocap10** [comment]
closed by #16
